### PR TITLE
automake: switch to xz tarball

### PR DESCRIPTION
**Pull request by diizzyy**
Maintainer: @xypron 

Description:

Switch to xz tarball

Signed-off-by: Daniel Engberg <daniel.engberg.lists@pyret.net>

### CODE REVIEW

**Comment by xypron** [comment]
@diizzyy 
The commit message does not explain why we should change to an xz tarball.

Regards

Heinrich Schuchardt

**Comment by xypron** [comment]
@diizzyy 

Prerequisite
package/feeds/packages/perl/compile fails.
So I currently cannot test this patch.

Regards

Heinrich Schuchardt

**Comment by diizzyy** [comment]
Hi, it's basically to save bandwidth and space. Want me to add that? Perl compiles fine here on LEDE (trunk) fwiw.

File: devel/automake/Makefile [hunk 1]
```diff
@@ -9,11 +9,11 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=automake
 PKG_VERSION:=1.15
-PKG_RELEASE:=3
+PKG_RELEASE:=4
```
Comment (yousong) [inline devel/automake/Makefile]: Guess there is no need to change `PKG_RELEASE` here as there should be no change in package content.

**Comment by xypron** [comment]
@diizzyy 

Acked-by: Heinrich Schuchardt <xypron.glpk@gmx.de>

**Pull request merged**